query-node/schema.graphql

```diff
@@ -226,6 +338,8 @@ type VideoMedia @entity {
 
   locationEntity: MediaLocationEntity
 
+  publishedBeforeJoystream: PublishedBeforeJoystream
+
```

Comment:
I think the concept of two separate types `VideoMedia` and `Video` was relic of old content directory design. There is only one concept of a `Video` in the content directory. Is there reason to have these two types still?

Reply:
Great, that simplifies the video structure in my eyes. Please check that the new structure is ok (and that assets are used correctly).